products/sle/main.pm

```diff
@@ -1272,6 +1282,14 @@ elsif (get_var("VIRT_AUTOTEST")) {
         loadtest "virt_autotest/host_upgrade_generate_run_file";
         loadtest "virt_autotest/host_upgrade_step2_run";
         loadtest "virt_autotest/reboot_and_wait_up_upgrade";
+        if (get_var('REPO_0_TO_INSTALL', '')) {
+            #After host upgrade, switch to version TARGET_DEVELOPING_VERSION if it is set
+            #Swtich to TARGET_DEVELOPING_VERSION
+            set_var('VERSION', get_var('TARGET_DEVELOPING_VERSION'));
+            bmwqemu::save_vars();
```

Comment:
these save_vars are unneeded - main.pm is evaluated in one run and after that the vars.json is written from isotovideo. No backend is running at this point anyway.

And as developers tend to copy&paste, I would consider them rather harmful.

Reply:
Ok, will remove.